### ISSUE TITLE
[A11Y] OJS Editorial View - Low contrast blue links when hovered or activated and missing underline

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#12650]
## Issue Description
Blue text links in the back end (e.g., in the "Start a Submission" form) have insufficient color contrast against the white background, making them difficult to distinguish from static text.

## How to reproduce / Current behavior
1. Open the "Start a Submission" form in the OJS back end.
2. Locate the blue text links within the instructions.
3. Move the keyboard focus or hover it with mouse pointer
4. Observe the lack of contrast and missing underlines.

## What application are you using?
OJS version 3.5.0

## Stack used 
- Device: Desktop
- OS: Windows/macOS
- Browser: Chrome browser

## Testing method
- Method: Manual test

## Action
Increase the color contrast of the blue links to at least 4.5:1 for every state(hover, click, focus) and consider adding an underline to identify them as interactive.

### Screenshot

<img width="1505" height="433" alt="link in the submission showing low contrast ratio" src="https://github.com/user-attachments/assets/98c491e3-b12c-42ac-8ae3-936d5f4e09d6" />

### WCAG Success Criteria
1.4.3 Contrast (Minimum) (AA)
